lib/components/WithdrawUI.jsx

```diff
@@ -44,7 +44,7 @@ const handleWithdrawSubmit = async (
   // TX overrides
   params.push({ gasLimit: 800000 })
 
-  await sendTx(setTx, provider, contractAddress, CompoundPrizePoolAbi, method, params, 'Withdraw')
+  await sendTx(setTx, provider, contractAddress, PrizePoolAbi, method, params, 'Withdraw')
```

Comment:
Are these ABIs interchangeable between 3.0.1, 3.1.0 and stake vs. compound prize pool?

Reply:
As far as I can tell, yes. All of the methods we use here (and other places I switched it) are implemented on the PrizePool contract, not the individual compound or staking contracts.